homework2/sass/footer.scss

```diff
@@ -0,0 +1,200 @@
+@import "common";
+
+footer {
+  background-color: $color-footer-back;
+  font-family: "Aclonica", sans-serif;
+  font-weight: 400;
+
+  .wrapper {
+    .container {
+      nav {
+        font-family: "Acme", sans-serif;
+        padding: 17px;
+
+        .footer-nav {
+          li {
+            list-style-type: none;
+            display: flex;
+
+            a {
+              text-decoration: none;
+              text-transform: uppercase;
+              @include fontProp(12px, 24px, $color-secondary-text);
+              letter-spacing: 2px;
+              padding: 6px;
+
+              &:hover {
+                @include fontProp(14px, 24px, #333);
+              }
+            }
+
+            &:last-child {
+              display: flex;
+              flex-direction: column;
+              text-transform: uppercase;
+              @include fontProp(12px, 24px, $color-secondary-text);
+              letter-spacing: 2px;
+              padding-left: 6px;
+
+              a {
+                padding: 0 0 6px;
+                @include fontProp(16px, 24px, $color-secondary-text);
+
+                &:hover {
+                  @include fontProp(18px, 24px, #333);
+                }
+              }
+            }
+          }
+        }
+      }
+
+      .contact-container {
+        .contact {
+          border-top: 2px solid $color-link;
```

Comment:
Why do you use link relatet variable for `border-top`?

Reply:
Done